### DISCUSSION TITLE
Как заблокировать стрелку возврата в таблицу цикла из расчетных таблиц для пользователей?

### DISCUSSION THREAD

**Discussion by IgorVolod** [totumonline/totum-ru-issues-and-discussions#622] [Как это сделать?]
Добрый день! Перекопал мануалы. не нашел ответа на этот вопрос. 
![стрелка](https://github.com/user-attachments/assets/8aaa2f3e-fb32-41e6-b876-83eef5e426d8)
Пользователю не нужно видеть таблицу циклов, он заполняет только расчетные таблицы. 
или, как вариант, клик по этой стрелке не должен открывать таблицу циклов, а должен выполнять редирект на модальное окно временной таблицы.

**Comment by erpic-ru** [comment]
Добрый день! Если прям в лоб решать, то закройте доступ к строкам таблицы циклов, например, через префильтр и в коде форматирования таблицы сделайте редирект всем или по условию, например:

```
f1=:setTableFormat(condition: $#nr!=1; tablehtml: $redirect;)
	redirect: str`'<meta http-equiv="refresh" content="0;URL=https://ссылка" />'`
	
```

**Reply by IgorVolod** [reply]
Да, редирект заработал, для всех ролей кто не создатель.  редирект отрывает временную таблицу, да, в виде обычной страницы, а она запилена под модаль.  ее бы в модальном окне открыть в ifarme...
